contexts/SubscriptionContext/src/UseCases/AddSubscription/AddSubscriptionUseCase.php

```diff
@@ -82,13 +84,15 @@ private function newMailAddressFromSubscription( Subscription $subscription ): E
 	}
 
 	private function createSubscriptionFromRequest( SubscriptionRequest $subscriptionRequest ): Subscription {
-		$request = new Subscription();
+		$subscription = new Subscription();
 
-		$request->setAddress( $this->addressFromSubscriptionRequest( $subscriptionRequest ) );
-		$request->setEmail( $subscriptionRequest->getEmail() );
-		$request->setConfirmationCode( random_bytes( 16 ) ); // No need to use uuid library here
+		$subscription->setAddress( $this->addressFromSubscriptionRequest( $subscriptionRequest ) );
+		$subscription->setEmail( $subscriptionRequest->getEmail() );
+		$subscription->setTracking( $subscriptionRequest->getTracking() );
+		$subscription->setSource( $subscriptionRequest->getSource() );
+		$subscription->setConfirmationCode( random_bytes( self::CONFIRMATION_CODE_BYTES ) ); // No need to use uuid library here
```

Comment:
This const name is a bit misleading; I thought a collection of bytes was being passed in here until I looked at the other side of the diff